v2.1/admin-ui-access-and-navigate.md

```diff
@@ -9,18 +9,22 @@ The built-in Admin UI gives you essential metrics about a cluster's health, such
 
 ## Access the Admin UI
 
-You can access the Admin UI from any node in the cluster.
+For insecure clusters, any user can access and view the Admin UI. For secure clusters, only authorized users can [access and view the Admin UI](#accessing-the-admin-ui-for-a-secure-cluster).
```

Comment:
maybe change "any user" to "anyone" to emphasize that there's no authentication at all?

Reply:
Done.